lab-andrew/html/index.html

```diff
@@ -0,0 +1,31 @@
+<!DOCTYPE html>
+<html ng-app="cowsayApp">
+  <head>
+    <title>Cowsay trolls</title>
+    <link rel="stylesheet" href="bundle.css" type="text/css"/>
+    <script src="bundle.js"></script>
+  </head>
+  <body>
+    <main ng-controller="CowsayController">
+      <form>
```

Comment:
Your form logic isn't quite right. To clarify what the form is doing I would suggest using the ng-submit directive in the form tag here. Add ng-submit="finalize()" in the form tag, and remove the ng-click from below.